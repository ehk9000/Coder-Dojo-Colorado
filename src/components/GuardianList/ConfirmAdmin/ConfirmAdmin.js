import React from 'react';

function ConfirmAdmin() {
  return (
    <div>

    </div>
  );
}

export default ConfirmAdmin;
