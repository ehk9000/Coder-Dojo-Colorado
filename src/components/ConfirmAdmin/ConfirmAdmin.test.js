import React from 'react';
import { shallow } from 'enzyme';
import ConfirmAdmin from './ConfirmAdmin';

describe('ConfirmAdmin', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ConfirmAdmin />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
