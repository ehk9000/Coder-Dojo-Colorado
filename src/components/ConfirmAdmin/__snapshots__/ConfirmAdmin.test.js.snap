// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmAdmin should match snapshot 1`] = `<div />`;
